src/@next/Pagination/PaginationStyle.tsx

```diff
@@ -15,6 +15,12 @@ export const SimplePaginationStyledNav = styled(StyledNav)`
   gap: 4px;
 `;
 
+export const IconPaginationStyledNav = styled(StyledNav)`
+  gap: 0px;
+  align-items: center;
+  column-gap: 1px;
+`;
+
 export const StyledPageButton = styled.button<PageButtonProps>`
   cursor: pointer;
   height: 32px;
```

Comment:
@davidzzz just realized that the size is incorrect either. The buttons should be 36px x 36px instead of 32px x 32px